components/Button.tsx

```diff
@@ -0,0 +1,44 @@
+// components/Hello.tsx
+import React from 'react';
+import {StyleSheet, Text, TouchableOpacity} from 'react-native';
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+
+export interface ButtonProps {
+    label: String
+}
+
+export class Button extends React.Component<ButtonProps> {
+    constructor(props: ButtonProps) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <TouchableOpacity style={styles.button}>
+                <Text style={styles.text}>{this.props.label}</Text>
```

Comment:
No FE, nós nomeamos os estilos sempre com o prefixo da classe, até para encapsular melhor o componente... Button e ButtonText
(No futuro, qdo vc for usar Styled-components, colocamos até o sufixo _Styled para indicar que é um styled component)

Reply:
Alterei os alias para o style destes componentes e do FormTextInput também, para ficar padronizado.